.github/workflows/codequality.yml

```diff
@@ -0,0 +1,40 @@
+name: codequality
+on: [pull_request]
+jobs:
+  golangci-lint:
+    name: golangci-lint, is a fast Go linters runner
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: golangci-lint
+        uses: reviewdog/action-golangci-lint@v1.10.1
```

Comment:
What is the difference between `reviewdog/action-golangci-lint` and the `golangci/golangci-lint-action` we have in `lint.yml`?

Reply:
i can remove this, i missed that we had a seperate lint action.